src/frontend/src/components/ChangeRequestRow.tsx

```diff
@@ -12,9 +12,10 @@ interface ChangeRequestRowProps {
   title: string;
   changeRequests: ChangeRequest[];
   noChangeRequestsMessage: string;
+  flexWrap?: string;
```

Comment:
Change this to a boolean and have it be wrap if true and nowrap if false. Just make sure to update the change request row props in the change request overview page with this approach